orcid-model/src/main/java/org/orcid/jaxb/model/record_rc2/PersonExternalIdentifier.java

```diff
@@ -119,6 +119,8 @@ public void setPutCode(Long putCode) {
         this.putCode = putCode;
     }
     public Visibility getVisibility() {
+        if (visibility == null)
```

Comment:
What does it show in the db, it should always be explicit? @wjrsimpson any thoughts?